ros2_generate_interface_docs/package.xml

```diff
@@ -8,6 +8,11 @@
   </description>
   <maintainer email="ahcorde@osrfoundation.org">Alejandro Hernández</maintainer>
   <license>Apache License 2.0</license>
+  
+  <exec_depend>python3-argparse</exec_depend>
```

Comment:
This is neither a valid `rosdep` key nor is any dependency needed for this. In Python 3 `argparse` is "built in".